docassemble/assemblylinewizard/generator_constants.py

```diff
@@ -80,17 +73,10 @@ class Object(object):
   'interested_party': 'interested_parties',
 }
 
-generator_constants.RESERVED_PRIMITIVE_PLURALIZERS_MAP = {
-  'docket_numbers': 'docket_numbers' 
-}
-
-generator_constants.RESERVED_PLURALIZERS_MAP = {
-  **generator_constants.RESERVED_PERSON_PLURALIZERS_MAP, 
-  **generator_constants.RESERVED_PRIMITIVE_PLURALIZERS_MAP,
-  **{
-    'court': 'courts', # for backwards compatibility 
-  }
-}
+generator_constants.RESERVED_PLURALIZERS_MAP = {** generator_constants.RESERVED_PERSON_PLURALIZERS_MAP, **{
+  'court': 'courts', # for backwards compatibility
+  'docket_numbers': 'docket_numbers',  
+}}
```

Comment:
You should rebase, or we're gonna have some merges that overwrite fixes already in master.  Or I guess you merge, but I just always have a lot more trouble with merges.